README.md

```diff
@@ -14,52 +14,55 @@ https://user-images.githubusercontent.com/49591562/149321073-ea311b45-3d7a-4d35-
 
-### <span style="font-family: Calibri, sans-serif;">Ausführung</span>
+### <span style="font-family: Calibri, sans-serif;">Execution</span>
 
-<span style="font-family: Calibri, sans-serif;">_Hinweis: Während das Programm arbeitet_ _am besten_ _keine der Tabellen öffnen_</span>
+<span style="font-family: Calibri, sans-serif;">_Note: While the program is working best do not open any of the tables_</span>
 
-##### <span style="font-family: Calibri, sans-serif;">Zu Software-Dateien navigieren</span>
+##### <span style="font-family: Calibri, sans-serif;">Navigate to software files</span>
 
-1.  <span style="font-family: Calibri, sans-serif;">"Eingabeaufforderung" (CMD) öffnen</span>
+1.  <span style="font-family: Calibri, sans-serif;">Open "Command Prompt" (CMD).</span>
 
-    1.  <span style="font-family: Calibri, sans-serif;">Windows-Taste + R drücken > "cmd" eingeben > Enter drücken</span>
+    1.  <span style="font-family: Calibri, sans-serif;">Press Windows key + R > type "cmd" > press Enter</span>
 
-2.  <span style="font-family: Calibri, sans-serif;">In CMD zu Pfad navigieren, in dem sich Crawler-Dateien ("plzCrawler.py" & "plzMaker.py") und das requirements.txt befinden</span>
-    1. <span style="font-family: Calibri, sans-serif;">Text nach folgendem Muster eingeben:  
+2.  <span style="font-family: Calibri, sans-serif;">Navigate in CMD to path where crawler files ("plzCrawler.py" & "plzMaker.py") and the requirements.txt are located</span>
+    1. <span style="font-family: Calibri, sans-serif;">Enter text according to the following pattern:  
         cd C:\Users\Username\Documents\plzcrawler</span>
 
-##### <span style="font-family: Calibri, sans-serif;">Virtuelle Umgebung (venv) erstellen</span>
+##### <span style="font-family: Calibri, sans-serif;">Create virtual environment (venv).</span>
 
-1.  <span style="font-family: Calibri, sans-serif;">Folgenden Text in CMD eingeben & Eingabetaste drücken (um virtuelle Umgebung mit dem Namen "venv" zu erstellen):  
-    py -m venv venv</span>
+1.  <span style="font-family: Calibri, sans-serif;">Type the following text in CMD & press Enter (to create a virtual environment named "venv"):
+     py -m venv venv</span>
     
-##### <span style="font-family: Calibri, sans-serif;">venv aktivieren</span>
+##### <span style="font-family: Calibri, sans-serif;">activate venv</span>
 
-1.  <span style="font-family: Calibri, sans-serif;">Folgenden Text in CMD eingeben & Eingabetaste drücken:  
-    venv\Scripts\activate.bat</span>
+1.  <span style="font-family: Calibri, sans-serif;">Type the following text in CMD & press Enter:
+     venv\Scripts\activate.bat</span>
 
-##### <span style="font-family: Calibri, sans-serif;">**Requirements.txt in venv installieren**</span>
+##### <span style="font-family: Calibri, sans-serif;">**Requirements.txt in venv to install**</span>
```

Comment:
you would rather say "install libraries from requirements.txt in venv".
Please adjust this.